[openusd_internal] Fix mis-use of system include path

This bug fix should be upstreamed.

--- pxr/usd/usdUtils/validators.cpp
+++ pxr/usd/usdUtils/validators.cpp
@@ -5,7 +5,7 @@
 // https://openusd.org/license.
 //
 
-#include <pxr/usd/sdf/fileFormat.h>
+#include "pxr/usd/sdf/fileFormat.h"
 #include "pxr/usd/usd/validationError.h"
 #include "pxr/usd/usd/validationRegistry.h"
 #include "pxr/usd/usdUtils/validatorTokens.h"
